test/Microsoft.Azure.Functions.Worker.Extensions.OpenApi.Tests/OpenApiHttpTriggerContextTests.cs

```diff
@@ -90,11 +91,34 @@ public async Task Given_Type_When_Initiated_Then_It_Should_Return_OpenApiConfigu
             options.Servers.Count.Should().Be(0);
         }
 
+        [TestMethod]
+        public async Task Given_CustomType_When_Initiated_Then_It_Should_Return_CustomOpenApiConfigurationOptions()
+        {
+            var customConfigurationOptions = new DefaultOpenApiConfigurationOptions
+            {
+                Info = new Microsoft.OpenApi.Models.OpenApiInfo
+                {
+                    Title = "A test title",
+                    Version = "2.0.0"
+                }
+            };
+
+            var location = new FileInfo(Assembly.GetExecutingAssembly().Location).Directory.FullName;
+            var context = new OpenApiHttpTriggerContext(customConfigurationOptions);
+
+            var options = (await context.SetApplicationAssemblyAsync(location, false))
+                                        .OpenApiConfigurationOptions;
+
+            options.Info.Version.Should().Be("2.0.0");
+            options.Info.Title.Should().Be("A test title");
+            options.Servers.Count.Should().Be(0);
+        }
+
         [TestMethod]
         public async Task Given_Type_When_Initiated_Then_It_Should_Return_OpenApiCustomUIOptions()
         {
             var location = new FileInfo(Assembly.GetExecutingAssembly().Location).Directory.FullName;
-            var context = new OpenApiHttpTriggerContext();
+            var context = new OpenApiHttpTriggerContext(null);
```

Comment:
If we change the constructor parameter to be optional, we don't need to change this part.